test/test_testing.py

```diff
@@ -1145,6 +1145,189 @@ def test_mismatching_values_msg(self):
                 fn()
 
 
+@unittest.skipIf(IS_FBCODE or IS_SANDCASTLE, "Not all sandcastle jobs support CSC testing")
```

Comment:
Just confirming: is this true or did you just copy-paste it from the CSR test case?

Reply:
It was just a copy-paste. I would not expect that CSR and CSC tests would behave differently.